docs/sources/getting-started/getting-started-elasticsearch.md

```diff
@@ -0,0 +1,107 @@
++++
+title = "With Grafana and Elasticsearch"
+description = "Guide for getting started with Grafana"
+keywords = ["grafana", "intro", "guide", "started", "Elastic", "Elasticsearch"]
+aliases = ["/docs/grafana/latest/guides/gettingstarted","/docs/grafana/latest/guides/getting_started"]
+weight = 400
++++
+
+# Getting started with Grafana and Elasticsearch
+
+Elasticsearch is a popular open source search and analytics engine for which Grafana provides out-of-the-box support. This topic walks you through the steps to create your first Elasticsearch backed dashboard in grafana to display any kind of data stored in your Elasticsearch database.
+
+## Step 1. Install Grafana and build your first dashboard
+
+Use the instructions in [Getting started with Grafana]({{< relref "getting-started.md" >}}) to:
+
+- Install Grafana.
+- Log in to Grafana.
+- Create your first dashboard.
+
+## Step 2. Download and install Elasticsearch
+
+Elasticsearch can be installed on many different operating systems. Refer to the [Installing Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/install-elasticsearch.html) for a complete list of all available options.
+
+Alternately you can install Elasticsearch using the resources available in [grafana/grafana](https://github.com/grafana/grafana) GitHub repository (recommended). Here you will find a collection of supported data sources, including Elasticsearch, along with test data and pre-configured dashboards for use.
```

Comment:
I would maybe say here something like - if they want to play/explore elastic data source and grafana, they can use `devenv/docker/blocs/elastic` to run elastic data source with random data . But I wouldn't say that you are installing Elasticsearch and that it is recommended.

Reply:
Good point. I actually copied this from other getting started guides we have, but I guess what was intended is that this is the recommended method to get started and play around, not to have a stable and production-ready system.
@oddlittlebird thoughts?

Reply:
Agree with you both. Will also look forward to Diana's opinion.

Reply:
You mean the dev environment? I think this guide is more for people wanting to actually use Elasticsearch with Grafana, so actual users. We have developer content in the Developers section. I would not reference the devenv here.